src/matter/interaction/InteractionMessages.ts

```diff
@@ -173,3 +173,46 @@ export const TlvInvokeResponse = TlvObject({
 export const TlvTimedRequest = TlvObject({
     timeout: TlvField(0, TlvUInt16),
 });
+
+/**
+ * @see {@link MatterCoreSpecificationV1_0} 10.6.6
+ */
+export const TlvWriteRequest = TlvObject({
+    suppressResponse: TlvOptionalField(0, TlvBoolean),
+    timedRequest: TlvOptionalField(1, TlvBoolean),
+    writeRequests: TlvField(2, TlvArray(TlvObject({
+        dataVersion: TlvOptionalField(0, TlvUInt32),
+        path: TlvField(1, TlvList({
```

Comment:
Extract this a sub-tlv structure since this is repeated

Reply:
hehe, I did already in one of the other PRs, so yes already existing. will adjust